bin/lib/docker.sh

```diff
@@ -1,5 +1,41 @@
 #! /usr/bin/env bash
 
+#######################################
+# Starts a bash shell in the civiform-shell container
+#######################################
+function docker::run_shell_container() {
+  # Allocate a TTY for better output even though not strictly needed.
+  docker-compose \
+    -f docker-compose.yml \
+    -f docker-compose.dev.yml \
+    -f docker-compose.shell.yml \
+    up civiform \
```

Comment:
yeah i guess im confused why we use civiform arg here and odn't use it other places?